.github/workflows/test.yml

```diff
@@ -173,9 +173,6 @@ jobs:
         run: |
           docker-compose -f .github/docker-compose.yaml up -d
 
-          # Wait for installation
-          sleep 40
```

Comment:
Based on the logs, it looks like the worker image was still installing packages when the logs got printed, but the pytest still succeeded. Maybe we were lucky that the pytests started after that started up?

Reply:
Yeah that's what I assume to be the case here? I'd like a more reliable way of checking that the cluster has been successfully initialized before jumping to the next step - going to explore pinging the worker/scheduler logs to accomplish this